### PR TITLE
Changes:

### LINKED ISSUE #52: Performance of MetadataTriggerHandler class is slower due to too much usage of if-else blocks.
## Steps to Reproduce the Problem

1. Perform a DML update and check the Execution Performance Time. 

## Specifications

- Version:1.0
- Platform: macOS

### PR DESCRIPTION
**Pull request by bishwambharsen**
- Reduced cognitive complexity and improved performance of the MetadataTriggerHandler class by replacing if else blocks with switch-case statements.
- Upgraded prettier version in package.json
-Added .pmdCache to .gitignore

Fixes #52

> It's a good idea to open an issue first for discussion.

- [x] Tests pass
- [x] Appropriate changes to README are included in PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -365,3 +365,7 @@ private static void beforeUpdate_test() {
 ```
 
 Notice how we performed _zero_ DML operations yet we were able to cover all of the logic of our class in this particular test. This can help save a lot of computational time and allow for much faster execution of Apex tests.
+
```
Comment (mitchspano) [inline README.md:368]: This is not really necessary for such a minor change. Let's omit the README updates.

**Review by mitchspano — CHANGES_REQUESTED** [review]
For such a small change with no client side visibility, I think we should just omit any updates to the readme. Otherwise, this is looking good - thank you for contributing!

File: README.md (hunk 1 above)
Comment (bishwambharsen) [reply README.md:368]: Sure @mitchspano :)

Comment (bishwambharsen) [reply README.md]: Done @mitchspano

**Review by mitchspano — APPROVED** [review]
Looks good!

**Pull request merged**